definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -15,21 +15,21 @@
     {"ID": "Respondant","CaseEventID": "FailedMediation","CaseFieldID": "respondents","ListElementCode": "partyName","EventElementLabel": "Defendant name","FieldDisplayOrder": 1,"DisplayContext": "READONLY"},
     {"ID": "Respondant","CaseEventID": "FailedMediation","CaseFieldID": "respondents","ListElementCode": "mediationFailedReason","EventElementLabel": "Mediation failed reason","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
     {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraOrderDirection","CaseFieldID": "otherDirections","EventElementLabel": "Select direction","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
-    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Additional instructions for document upload","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
-    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "otherDirectionHeaders","CaseFieldID": "otherDirections","EventElementLabel": "Direction header","EventHintText": "e.g. Upload contracts /Confirm documents are original","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
-    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "directionComment","CaseFieldID": "otherDirections","EventElementLabel": "Direction comment","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "OtherDirection","CaseEventID": "GenerateOrder","ListElementCode": "extraDocUploadList","CaseFieldID": "otherDirections","EventElementLabel": "Directions regarding expert","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
```

Comment:
is `EventHintText` mandatory even when empty?

Reply:
No idea. Let me find out!

Reply:
It is not.